client/css/form.css

```diff
@@ -204,18 +213,13 @@ form .inputControl {
   display: flex;
```

Comment:
Add 
justify-content: flex-start;
 gap: 1px;

so the error icon and message are align properly.